operator/build/molecule.Dockerfile

```diff
@@ -3,9 +3,10 @@ FROM centos:7
 RUN mkdir -p /home/molecule
 
 ENV HOME=/home/molecule
-ENV PYTHONUSERBASE=${HOME}/.bin
-RUN mkdir -p ${PYTHONUSERBASE}
-ENV PATH="${HOME}/.bin:${PATH}"
+ENV PYTHONUSERBASE=${HOME} \
+    PATH="${HOME}/bin:${PATH}"
```

Comment:
You sure this is right? When I did `pip install --user molecule==2.20.0.0a2` on my machine it put it in `$HOME/.local/bin`

Reply:
Yeah, the overridden PYTHONUSERBASE makes it just $HOME/bin

Reply:
Yeah, the overridden PYTHONUSERBASE makes it just $HOME/bin